Tests/Admin/AdminTest.php

```diff
@@ -2094,6 +2095,75 @@ public function testGetDataSourceIterator()
         $admin->getDataSourceIterator();
     }
 
+    public function testCircularChildAdmin()
+    {
+        $this->expectException(
+            'RuntimeException',
+            'Circular reference detected!'.
+            ' The child admin `sonata.post.admin.post` is already in the parent tree'.
+            ' of the `sonata.post.admin.comment` admin.'
```

Comment:
@Soullivaneuh Is it good to line-break exception messages like that? You [said](https://github.com/sonata-project/SonataAdminBundle/pull/4542#pullrequestreview-64263705) that exception messages should not be line-breaked, but I think it was because I used annotations before. Could you confirm?

Reply:
If you have time to kill, long bike-shedding thread about this: https://github.com/sonata-project/dev-kit/issues/218